cluster-provision/images/vm-image-builder/customize-image.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+set -exuo pipefail
+
+function cleanup() {
+  echo "[customize-image] cleanup"
+  rm -rf "$CLOUD_INIT_ISO"
+  virsh destroy "$DOMAIN_NAME" || true
+  virsh undefine "$DOMAIN_NAME" || true
+
+  if [ $? -ne 0 ]; then
+    rm -f "$NEW_IMAGE"
+  fi
+}
+
+IMAGE_PATH=$1
+NEW_IMAGE=$2
```

Comment:
Is this also a path, right?

Reply:
Yes this the path of the new qcow2 file that will be created

Reply:
So if be consistent and add "PATH" to this name as well.
Maybe you should call them SRC and DST (instead of NEW) for name consistency.

Reply:
How about `SOURCE_IMAGE` and `CUSTOMIZED_IMAGE`?